device/mellanox/x86_64-mlnx_msn2700-r0/plugins/psuutil.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python
```

Comment:
Exactly same file as msn2410. Please use a symbol link.

Reply:
Replaced to  a symbol link.